### DISCUSSION TITLE
Help with Install — “Releasing Pressure!” Brew switch not working

### DISCUSSION THREAD

**Discussion by stiffsteve** [Zer0-bit/gaggiuino#557] [Q&A]
Hello, just completed a v3.1 install. When I powered it on, the screen was flashing “Releasing Pressure!” repeatedly, but stopped after ~15 seconds. Strange, but OK. I went to Brew->Manual to do the machine fill, set the flow rate, hit the Brew switch, but nothing happened.

 Eventually I flipped it in to Steam mode, just to see, and the screen completely shut off. I tripled checked the switch wiring, unsure what might have went wrong? Thanks in advance!

**Comment by giusere95** [comment]
Hello, have the same problem here, did you manage to solve it by any chance?
